packages/flutter_app_badger/tizen/src/tizen_badge.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_badge.h"
+
+#include "log.h"
+
+bool TizenBadge::Initialize() {
+  char *app_id = nullptr;
+  int ret = app_get_id(&app_id);
+  if (ret != APP_ERROR_NONE) {
+    LOG_ERROR("app_get_id() failed with error %s.", get_error_message(ret));
```

Comment:
```suggestion
    LOG_ERROR("app_get_id() failed with error: %s", get_error_message(ret));
```
The string returned by `get_error_message` is expected to be a complete sentence that ends with a period.

Reply:
done